### ISSUE TITLE
loki-stack update (component version)

### ISSUE THREAD

**Issue by warmanton** [grafana/helm-charts#690]
Is any way to update loki, promtail and other component versions since loki-stack-2.4.1 ?
Or probably it is time to do new release of loki-stack  with fresh software ?

**Comment by ncolboc** [comment]
yes, it will be great! there is a PR but unfortunately has failed during lint-test step
https://github.com/grafana/helm-charts/pull/605

**Comment by nenych** [comment]
Any updates on this issue?
PR has been opened for more than 1 month.

**Comment by Nexus2k** [comment]
> From the user point of view: I think this change (I mean the PR which changes only the versions) can break some things.

Sound like a thing one could document and provide upgrade instructions for, no? Well looks like we'll now have to build our own chart to get the latest Loki version instead.

**Comment by dbpolito** [comment]
I was looking at this and everything seems outdated... is that really well maintained? 

I think at least needs to add a warning that it's not up to date and recommend not using it and installing all other components separately.

**Comment by Nexus2k** [comment]
I've ended up just deploying the loki and promtail charts individually instead.

**Comment by warmanton** [comment]
Few days ago I deployed new version of Loki via ansible+docker-compose and my old playbook failed because fresh version of Loki requires parameter `-ingester.wal-dir=/path/to/wal-dir`. After adding yet another mount point to docker-compose.yml and updating the `command:` Loki starts and works as expected. May be the helm chart should be fixed in the same way ?
I cannot try this by my self.

**Comment by Sheikh-Abubaker** [comment]
The loki-stack chart is deprecated and no longer maintained. Please migrate to the actively maintained [Loki](https://github.com/grafana/loki/tree/main/production/helm/loki) chart referenced in https://github.com/grafana/helm-charts/pull/3974, Closing this issue as not planned
